nautobot/ipam/views.py

```diff
@@ -713,6 +713,10 @@ def extra_post_save_action(self, obj, form):
             obj.locations.add(*form.cleaned_data["add_locations"])
         if form.cleaned_data.get("remove_locations", None):
             obj.locations.remove(*form.cleaned_data["remove_locations"])
+        if form.cleaned_data.get("add_vrfs", None):
+            obj.vrfs.add(*form.cleaned_data["add_vrfs"])
+        if form.cleaned_data.get("remove_vrfs", None):
+            obj.vrfs.remove(*form.cleaned_data["remove_vrfs"])
```

Comment:
Do we need to do anything special here to prevent assigning VRFs from the wrong namespace to prefixes in a different namespace? I don't think this implementation will call `VRFPrefixAssignment.clean()` for example.

Also, can we add some vrfs to the Prefix view test case bulk_edit_data to exercise this functionality?

Finally, shouldn't we similarly add `add_prefixes` and `remove_prefixes` fields to the `VRFBulkEditForm` as well?

Reply:
We don't need to handle, as its been handled by the `vrf_prefix_associated` signal: 
```python
@receiver(m2m_changed, sender=VRFPrefixAssignment)
def vrf_prefix_associated(sender, instance, action, reverse, model, pk_set, **kwargs):
    """
    Disallow adding Prefixes if the namespace doesn't match.
    """

    if action == "pre_add":
        prefixes = model.objects.filter(pk__in=pk_set).exclude(namespace=instance.namespace)
        if prefixes.exists():
            raise ValidationError({"prefixes": "Prefix must match namespace of VRF"})
```

Reply:
Ok, thanks for confirming. I see you updated `VRFBulkEditForm` as well now. Can we get some test coverage for these form fields?